### ISSUE TITLE
If a print is canceled while is heatingUp, printer will continue heatingUp

### ISSUE THREAD

**Issue by JulianGarciaCastillo** [AstroPrint/AstroBox#290]
If user cancel a print while the printer is in heatingUp state, the printer continues in heatingUp state until reboot it.

**Comment by FernandoGarcia** [comment]
Hi!
@julianCast @CoDanny 
I have some problem like this with 0.13.5.
The heating stops only after reach the target then heaters are turned off.
Best regards.

**Comment by CoDanny** [comment]
@FernandoGarcia Doesn't seem like the bug. Please open a new ticket.
